dependency/actions/compile/extensions-manifests/extensions-7.4.yml

```diff
@@ -0,0 +1,191 @@
+---
```

Comment:
@TisVictress will these extensions need to be updated ever?

Reply:
If so we should file an issue about that so we don't forget

Reply:
Based on the history of these files in the [binary-builder code](https://github.com/cloudfoundry/buildpacks-ci/blob/master/tasks/build-binary-new-cflinuxfs4/php8-base-extensions.yml), I think it doesn't need to be updated. Maybe @brayanhenao can correct me if I'm wrong.